### ISSUE TITLE
Javascript appears as reading content

### ISSUE THREAD

**Issue by micealgallagher** [cameron/squirt#131]
### Steps to reproduce

1) Use Chrome 33.0.1750.154
2) Navigate to http://www.chicagotribune.com/news/local/breaking/chi-bp-oil-spill-lake-michigan-20140327,0,7519350.story
3) Select the text from _"BP today more than doubled its maximum..."_ to _"...processing capacity will do no harm to Lake Michigan.”"_
4) Click on the [Squirt] bookmark

When the reading point passes _"an ArcelorMittal steel mill."_ javascript appears as being readable content.

**Comment by cameron** [comment]
thanks for the report!

**Comment by micealgallagher** [comment]
No problem.

If you need additional information just let me know.

**Comment by micealgallagher** [comment]
Also. I Love Squirt since I discovered it I use it everyday.

**Comment by gitprod** [comment]
also with firefox (latest, I presume) on linux mint.

Could not get it to read selected text (instead started at beginning every time).
Not wholly disappointed, as I want to study javascript, I'm sure some of the info was retained ;)

**Comment by gitprod** [comment]
website was [Christian Science monitor's story on Chinese drive for thorium-based reactors](http://www.csmonitor.com/Environment/Energy-Voices/2014/0328/Thorium-a-safer-nuclear-power)

**Comment by ghost** [comment]
It's also in blank new tab in Chrome (I have 34.0.1847.132 on Ubuntu)

**Comment by metakermit** [comment]
Yes, I also tried reading [a Medium](https://medium.com/@eteraz/the-death-of-the-urdu-script-9ce935435d90) article and was greeted by a bunch of markup and code gibberish. Probably needs to go through some Readability-style filter first.

Great work otherwise, keep it up!
